www/_posts/2017-04-21-cordova-7.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Apache Cordova 7.0.0"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that `Apache Cordova 7.0.0` has been released!
+
+Most notable changes include:
+* If a `package.json` does not exist in your project, it will be auto-created for you when `cordova prepare` is called.
+* When adding a platform or plugin, it will automatically save that platform or plugin to your `config.xml` and `package.json`. Details about platform and plugin versions are also automatically saved in `config.xml` and `package.json`. The `--save` flag is no longer required to save. Use `--nosave` to prevent saving to `config.xml` or `package.json`.
+* Fetch is now the default setting for fetching platforms. Fetch uses your system `npm` to `npm install` modules into your project. The `--fetch` flag is no longer required. Use the `--nofetch` flag to use the older cordova fetching method (pre `cordova@7`) and prevent using `npm install` to do the fetching.
+* After `cordova prepare` is run, `package.json` and `config.xml` should contain identical platforms and versions.  In case of conflicts, `package.json` is given precedence over `config.xml`. For example, suppose `package.json` contains `cordova-android@6.0.0` and `config.xml` contains `cordova-android@4.0.0`. After `cordova prepare` is run, `config.xml` and `package.json` will each contain only `cordova-android@6.0.0`.
+* Users can install any platform due to removing known platform check for platform API. For instance, users can now add custom platforms to their projects.
```

Comment:
~* Users can install any platform due to removing known platform check for platform API. For instance, users can now add custom platforms to their projects.~
* We have removed a requirement that `platform add` be called with a '_known platform_' and now simply expect that whatever you pass '_behaves like a known platform_'.  This will allow future custom platforms, and modified version of existing platforms.